sig-release/charter.md

```diff
@@ -96,7 +96,7 @@ Additional requirements:
 - [sigs.yaml] must be updated to include subproject information and OWNERS files with subproject chairs
 
 
-[KEP]: https://git.k8s.io/enhancements/keps/YYYYMMDD-kep-template.md
+[KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md
```

Comment:
How about linking to https://git.k8s.io/enhancements/keps ?

Reply:
The previous link was done to the template of a KEP. Surely, linking it to the whole repo would be very useful, but I'm not sure if we really need that.

Reply:
https://git.k8s.io/enhancements/keps is in fact the index of KEPs and also a page that explains what a KEP is; that's why I suggested linking there.

Reply:
Ahh yes, that makes sense. I'll do it, thanks!